acceptance/bundle/deploy/pipeline/recreate/databricks.yml.tmpl

```diff
@@ -22,13 +22,10 @@ resources:
 
   schemas:
     bar:
-      name: test-schema-{{.unique_id}}
+      name: test-schema-$UNIQUE_NAME
       catalog_name: ${var.catalog}
       comment: This schema was created from DABs
 
-include:
-  - "*.yml"
```

Comment:
I suppose you removed it because there is nothing to include? Weird that it did not complain before.